packages/frontend/pages/api/isValidAddress.ts

```diff
@@ -18,8 +20,16 @@ const handleRequest = async (req: NextApiRequest, res: NextApiResponse) => {
     )
 
     res.status(200).json({ valid: (data?.[0]?.rating ?? 'highRisk') !== 'highRisk' })
```

Comment:
Did you try with risky address? To my knowledge, if you don't return at this point code will continue and always return as true

That brings my 2nd point, Can we add tests to make sure this always works? Can mock the `chainalysis` requests for different cases